analyzer-comments/elm/common/no_unused/custom_type_constructor_args.md

```diff
@@ -0,0 +1,13 @@
+# custom type constructor arguments
+
+[comment]: # (This error is raised when a type constructor argument is unused)
+[comment]: # (requires 1 param:)
+[comment]: # (- definition: code extract with unused value)
+
+The type constructor argument indicated below is not used anywhere.
```

Comment:
Does this rule fire for phantom types? I assume not but would be good to be sure!

Reply:
I double checked and it does not fire. It fires only for constructor arguments.

Reply:
Cool, could you mention that somewhere, in the comment or something?